go/chat/boxer.go

```diff
@@ -620,6 +648,11 @@ func (b *Boxer) verify(data []byte, si chat1.SignatureInfo, prefix libkb.Signatu
 // This trusts the server for ctime, so a colluding server could use a revoked key and this check wouldn't notice.
 // Returns (validAtCtime, revoked, err)
 func (b *Boxer) ValidSenderKey(ctx context.Context, sender gregor1.UID, key []byte, ctime gregor1.Time) (found, validAtCTime bool, revoked *gregor1.Time, unboxErr UnboxingError) {
+	if b.testingValidSenderKey != nil {
+		b.log().Warning("Using TESTING jig. Not suitable for normal use.")
```

Comment:
Added that flag+panic which might work and in the worst case will cause tests to fail.